app/src/main/java/com/example/event_finder_capstone/activities/EventDetailsActivity.java

```diff
@@ -96,6 +96,25 @@ public boolean onDoubleTap(MotionEvent e) {
         getWindow().getDecorView().setOnTouchListener((view, event) -> detector.onTouchEvent(event));
     }
 
+    private void removeBookmark(String eventId) {
+        ParseQuery<Bookmark> query = ParseQuery.getQuery(Bookmark.class);
+        query.whereEqualTo(Bookmark.KEY_BOOKMARKED_EVENT_ID, eventId);
+        query.whereEqualTo(Bookmark.KEY_USER, ParseUser.getCurrentUser());
+        query.getFirstInBackground((bookmarks, e) -> {
+            if (e != null) {
+                Toast.makeText(EventDetailsActivity.this, "Failed to remove bookmark", Toast.LENGTH_LONG).show();
+            }
+            try {
+                bookmarks.delete();
+            } catch (com.parse.ParseException ex) {
+                ex.printStackTrace();
```

Comment:
🔍 Nit:  Presumably this will also fail to remove the bookmark in Parse and also want to let the user know.